bam-index/src/main/java/org/victorchang/IndexVersion.java

```diff
@@ -0,0 +1,26 @@
+package org.victorchang;
+
+public abstract class IndexVersion {
+    private IndexVersion() {
+    }
+
+    public static final IndexVersion LATEST = new IndexVersion() {
```

Comment:
Let's put a javadoc on this class explaining what I should do if I change the index format?
e.g. Is it enough to just bump the `getVersion` return value?